src/instruments/src/PFD/PFD.tsx

```diff
@@ -0,0 +1,138 @@
+import { A320Failure, FailuresConsumer } from '@flybywiresim/failures';
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
Why is this needed?

Reply:
It's the `FSComponent` import that is reproted as unused but needed. It's more or less the equivalent to `import 'react'` (https://microsoft.github.io/msfs-avionics-mirror/docs/getting-started/creating-your-first-component/#framework-imports). Maybe I need to adapt the eslint config so it properly recognizes it.